applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.cpp

```diff
@@ -435,20 +435,15 @@ void UPwBaseElement::CalculateAll(MatrixType&        rLeftHandSideMatrix,
     KRATOS_CATCH("")
 }
 
-double UPwBaseElement::CalculateIntegrationCoefficient(const GeometryType::IntegrationPointType& rIntegrationPoint,
-                                                       double detJ) const
-
-{
-    return mpStressStatePolicy->CalculateIntegrationCoefficient(rIntegrationPoint, detJ, GetGeometry());
-}
-
-std::vector<double> UPwBaseElement::CalculateIntegrationCoefficients(
+//----------------------------------------------------------------------------------------
```

Comment:
Could you remove this separator line? It's not really necessary and clutters the file a bit (I know we have it in other places, but we're planning to remove them)

Reply:
done